src/test/regress/bin/diff

```diff
@@ -29,13 +30,14 @@ then
     DIFF=/usr/bin/diff
 fi
 
-if test -z "$VANILLATEST"
+if test -z "${VANILLATEST:-}"
 then
-	sed -Ef $BASEDIR/normalize.sed < $file1 > $file1.modified
-	sed -Ef $BASEDIR/normalize.sed < $file2 > $file2.modified
-	$DIFF -w $args $file1.modified $file2.modified
+	sed -Ef $BASEDIR/normalize.sed < $file1 > "$file1.modified"
```

Comment:
`sed -i` can be used to modify files in-place, removing need for this `< > mv` dance